docs/scripts/utils.py

```diff
@@ -32,9 +32,15 @@ def adjust_pictures(
 
     if line_index is not None:
         line = lines[line_index]
-        light_line = line.replace("reference external", "reference external only-light")
+        light_line = line
+        if "reference external" in line:  # for replacing the banner
+            light_line = line.replace(
+                "reference external", "reference external only-light"
+            )
+        if "_images/full_lookup_light.svg" in line:  # for the example in the README
+            light_line = line.replace("img ", 'img class="only-light align-center" ')
         lines[line_index] = light_line
-        dark_line = light_line.replace("only-light", "only-dark")
+        dark_line = light_line.replace("light", "dark")
```

Comment:
question about the approach here, especially line 45 (cant comment on it):
to me seems the `dark_line` should be inserted, but the current version is overriding whatever line comes after line number `line_index`. Also if the match is in the very last line this would fail. Isnt this dangerous?

Reply:
True, will change that to properly insert.